dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.37/V2_37_18__New_Columns_in_Visualization.sql

```diff
@@ -0,0 +1,7 @@
+-- This script relates to the task https://jira.dhis2.org/browse/DHIS2-11347
+
+-- Rename column: "legend" to "serieskey".
+ALTER TABLE visualization RENAME COLUMN legend TO serieskey;
+
+-- Add new boolean flag related to legend.
+ALTER TABLE visualization ADD COLUMN IF NOT EXISTS legendhidekey BOOLEAN DEFAULT FALSE;
```

Comment:
This is the comment from Lars to DHIS2-7029: _We have a convention of not using default values. Instead 1) add column as nullable 2) set null values to false 3) add not null constraint to column._
If you do not need not null step 3 is redundant

Reply:
Alright, done.